pkg/operator/controllers/workaround/ifreload.go

```diff
@@ -17,11 +17,23 @@ import (
 const kubeNamespace = "openshift-azure-ifreload"
 
 type ifReload struct {
-	log          *logrus.Entry
-	cli          kubernetes.Interface
+	log *logrus.Entry
+
+	cli kubernetes.Interface
+
 	versionFixed *version.Version
 }
 
+func NewIfReload(log *logrus.Entry, cli kubernetes.Interface) Workaround {
+	verFixed, _ := version.ParseVersion("4.4.10")
+
+	return &ifReload{
+		log:          log,
+		cli:          cli,
+		versionFixed: verFixed,
+	}
+}
```

Comment:
the `New` function is directly after the type definition for all the other controllers.